frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -36,6 +36,7 @@ const tableColumnClasses = [
   classNames('col-lg-2', 'col-md-2', 'col-sm-4', 'hidden-xs'),
   classNames('col-lg-3', 'col-md-3', 'hidden-sm', 'hidden-xs'),
   classNames('col-lg-3', 'col-md-3', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-3', 'col-md-3', 'hidden-sm', 'hidden-xs'),
```

Comment:
The column classes no longer add up to 12.

We should consider removing Bootstrap grid here as we've done in other places (for instance, https://github.com/openshift/console/blob/master/frontend/public/components/pod.tsx#L97-L108)

Reply:
@spadgett There is another that resolved the col classes issue. - https://github.com/openshift/console/pull/4482

Reply:
@cyril-ui-developer Ack, thanks. I did just open a related bug

https://bugzilla.redhat.com/show_bug.cgi?id=1834361